2024-09-10 14:26:09,443 | WARNING | Неверный тип данных для объекта Runner
Traceback (most recent call last):
  File "C:\Divey\PythonProjects\строки\test.py", line 72, in test_run
    gump = Runner(True, 5)
           ^^^^^^^^^^^^^^^
  File "C:\Divey\PythonProjects\строки\test.py", line 13, in __init__
    raise TypeError(f'Имя может быть только строкой, передано {type(name).__name__}')
TypeError: Имя может быть только строкой, передано bool
2024-09-10 14:26:09,444 | WARNING | Неверная скорость для Runner
Traceback (most recent call last):
  File "C:\Divey\PythonProjects\строки\test.py", line 61, in test_walk
    forest = Runner('Forest', -5)
             ^^^^^^^^^^^^^^^^^^^^
  File "C:\Divey\PythonProjects\строки\test.py", line 18, in __init__
    raise ValueError(f'Скорость не может быть отрицательной, сейчас {speed}')
ValueError: Скорость не может быть отрицательной, сейчас -5
